src/pages/_app.js

```diff
@@ -120,7 +128,9 @@ function App({ Component, pageProps }) {
                 flexDirection="column"
                 flexGrow={1}
                 overflow="scroll"
-                className={showBackground ? classes.background : null}
+                className={`${background === 'none' ? classes.backgroundNone : ''} 
+                ${background === 'blue' ? classes.backgroundBlue : ''}
+                ${background === 'peach' ? classes.backgroundPeach : ''}`}
```

Comment:
Potentially silly question; could you set the state to be the class name and then just do something like `className={backgroundClass}`?

Reply:
Tried this before but with fresh eyes I think I might be able to do something! Let me try & get back to you

Reply:
Made better!